packages/frontend-2/components/tour/Onboarding.vue

```diff
@@ -4,41 +4,20 @@
   >
     <TourSegmentation v-if="showSegmentation && step === 0" @next="step++" />
     <TourSlideshow v-if="step === 1" @next="step++" />
```

Comment:
I think the step number maybe can be removed? I think one boolean should be enough? `showSlideshow` or something after `TourSegmentation` emitted `next`